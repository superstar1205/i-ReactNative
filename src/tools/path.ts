import { GluegunFilesystem } from "gluegun"
import * as path from "path"

export const BOILERPLATE_PATH = path.join(__dirname, "../..", "/boilerplate")

export function createGetAllFilePaths(filesystem: GluegunFilesystem) {
  /**
   * Recursively get all file paths in a directory
   */
  return function getAllFilePaths(dir: string): string[] {
    const files = filesystem.list(dir)
    const filePaths = files
      .map((file) => `${dir}/${file}`)
      .filter((file) => filesystem.isFile(file))
    const subdirs = files.filter((file) => filesystem.isDirectory(`${dir}/${file}`))
    const subdirPaths = subdirs.map((subdir) => getAllFilePaths(`${dir}/${subdir}`))
    return [...filePaths, ...subdirPaths.flat()]
  }
}
