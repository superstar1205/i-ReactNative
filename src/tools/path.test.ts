import { filesystem } from "gluegun"
import { BOILERPLATE_PATH, createGetAllFilePaths } from "./path"

describe("paths", () => {
  describe("getAllFilePaths", () => {
    it(`should return expected paths in '~/boilerplate'`, () => {
      const getAllFilePaths = createGetAllFilePaths(filesystem)
      const paths = getAllFilePaths(BOILERPLATE_PATH).map((path) => path.replace(process.cwd(), ""))

      expect(paths).toMatchSnapshot()
    })
  })
})
