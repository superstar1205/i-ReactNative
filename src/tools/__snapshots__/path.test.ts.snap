// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`paths getAllFilePaths should return expected paths in '~/boilerplate' 1`] = `
Array [
  "/boilerplate/.gitignore",
  "/boilerplate/.gitignore.template",
  "/boilerplate/.npmrc",
  "/boilerplate/.prettierignore",
  "/boilerplate/App.js",
  "/boilerplate/README.md",
  "/boilerplate/app.json",
  "/boilerplate/babel.config.js",
  "/boilerplate/index.js",
  "/boilerplate/jest.config.js",
  "/boilerplate/metro.config.js",
  "/boilerplate/package.json",
  "/boilerplate/react-native.config.js",
  "/boilerplate/tsconfig.json",
  "/boilerplate/webpack.config.js",
  "/boilerplate/yarn.lock",
  "/boilerplate/android/build.gradle",
  "/boilerplate/android/gradle.properties",
  "/boilerplate/android/gradlew",
  "/boilerplate/android/gradlew.bat",
  "/boilerplate/android/settings.gradle",
  "/boilerplate/android/app/BUCK",
  "/boilerplate/android/app/_BUCK",
  "/boilerplate/android/app/build.gradle",
  "/boilerplate/android/app/build_defs.bzl",
  "/boilerplate/android/app/debug.keystore",
  "/boilerplate/android/app/proguard-rules.pro",
  "/boilerplate/android/app/src/debug/AndroidManifest.xml",
  "/boilerplate/android/app/src/debug/java/com/helloworld/ReactNativeFlipper.java",
  "/boilerplate/android/app/src/main/AndroidManifest.xml",
  "/boilerplate/android/app/src/main/java/com/helloworld/.gitignore",
  "/boilerplate/android/app/src/main/java/com/helloworld/MainActivity.java",
  "/boilerplate/android/app/src/main/java/com/helloworld/MainApplication.java",
  "/boilerplate/android/app/src/main/java/com/helloworld/newarchitecture/MainApplicationReactNativeHost.java",
  "/boilerplate/android/app/src/main/java/com/helloworld/newarchitecture/components/MainComponentsRegistry.java",
  "/boilerplate/android/app/src/main/java/com/helloworld/newarchitecture/modules/MainApplicationTurboModuleManagerDelegate.java",
  "/boilerplate/android/app/src/main/jni/Android.mk",
  "/boilerplate/android/app/src/main/jni/MainApplicationModuleProvider.cpp",
  "/boilerplate/android/app/src/main/jni/MainApplicationModuleProvider.h",
  "/boilerplate/android/app/src/main/jni/MainApplicationTurboModuleManagerDelegate.cpp",
  "/boilerplate/android/app/src/main/jni/MainApplicationTurboModuleManagerDelegate.h",
  "/boilerplate/android/app/src/main/jni/MainComponentsRegistry.cpp",
  "/boilerplate/android/app/src/main/jni/MainComponentsRegistry.h",
  "/boilerplate/android/app/src/main/jni/OnLoad.cpp",
  "/boilerplate/android/app/src/main/res/drawable/rn_edit_text_material.xml",
  "/boilerplate/android/app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml",
  "/boilerplate/android/app/src/main/res/mipmap-hdpi/bootsplash_logo.png",
  "/boilerplate/android/app/src/main/res/mipmap-hdpi/ic_launcher.png",
  "/boilerplate/android/app/src/main/res/mipmap-hdpi/ic_launcher_background.png",
  "/boilerplate/android/app/src/main/res/mipmap-hdpi/ic_launcher_foreground.png",
  "/boilerplate/android/app/src/main/res/mipmap-hdpi/ic_launcher_round.png",
  "/boilerplate/android/app/src/main/res/mipmap-mdpi/bootsplash_logo.png",
  "/boilerplate/android/app/src/main/res/mipmap-mdpi/ic_launcher.png",
  "/boilerplate/android/app/src/main/res/mipmap-mdpi/ic_launcher_background.png",
  "/boilerplate/android/app/src/main/res/mipmap-mdpi/ic_launcher_foreground.png",
  "/boilerplate/android/app/src/main/res/mipmap-mdpi/ic_launcher_round.png",
  "/boilerplate/android/app/src/main/res/mipmap-xhdpi/bootsplash_logo.png",
  "/boilerplate/android/app/src/main/res/mipmap-xhdpi/ic_launcher.png",
  "/boilerplate/android/app/src/main/res/mipmap-xhdpi/ic_launcher_background.png",
  "/boilerplate/android/app/src/main/res/mipmap-xhdpi/ic_launcher_foreground.png",
  "/boilerplate/android/app/src/main/res/mipmap-xhdpi/ic_launcher_round.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxhdpi/bootsplash_logo.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxhdpi/ic_launcher.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_background.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxhdpi/ic_launcher_round.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxxhdpi/bootsplash_logo.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_background.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_foreground.png",
  "/boilerplate/android/app/src/main/res/mipmap-xxxhdpi/ic_launcher_round.png",
  "/boilerplate/android/app/src/main/res/values/colors.xml",
  "/boilerplate/android/app/src/main/res/values/strings.xml",
  "/boilerplate/android/app/src/main/res/values/styles.xml",
  "/boilerplate/android/gradle/wrapper/gradle-wrapper.jar",
  "/boilerplate/android/gradle/wrapper/gradle-wrapper.properties",
  "/boilerplate/app/app.tsx",
  "/boilerplate/app/components/AutoImage.tsx",
  "/boilerplate/app/components/Button.tsx",
  "/boilerplate/app/components/Header.tsx",
  "/boilerplate/app/components/Icon.tsx",
  "/boilerplate/app/components/ListItem.tsx",
  "/boilerplate/app/components/Screen.tsx",
  "/boilerplate/app/components/Text.tsx",
  "/boilerplate/app/components/TextField.tsx",
  "/boilerplate/app/components/index.ts",
  "/boilerplate/app/components/bullet-item/bullet-item.tsx",
  "/boilerplate/app/components/checkbox/checkbox.props.ts",
  "/boilerplate/app/components/checkbox/checkbox.tsx",
  "/boilerplate/app/components/switch/switch.props.ts",
  "/boilerplate/app/components/switch/switch.tsx",
  "/boilerplate/app/config/config.base.ts",
  "/boilerplate/app/config/config.dev.ts",
  "/boilerplate/app/config/config.prod.ts",
  "/boilerplate/app/config/index.ts",
  "/boilerplate/app/i18n/ar.ts",
  "/boilerplate/app/i18n/en.ts",
  "/boilerplate/app/i18n/i18n.ts",
  "/boilerplate/app/i18n/index.ts",
  "/boilerplate/app/i18n/ko.ts",
  "/boilerplate/app/i18n/translate.ts",
  "/boilerplate/app/models/AuthenticationStore.ts",
  "/boilerplate/app/models/Episode.test.ts",
  "/boilerplate/app/models/Episode.ts",
  "/boilerplate/app/models/EpisodeStore.ts",
  "/boilerplate/app/models/RootStore.ts",
  "/boilerplate/app/models/index.ts",
  "/boilerplate/app/models/helpers/get-root-store.ts",
  "/boilerplate/app/models/helpers/setup-root-store.ts",
  "/boilerplate/app/models/helpers/use-stores.ts",
  "/boilerplate/app/models/helpers/with-set-prop-action.ts",
  "/boilerplate/app/navigators/AppNavigator.tsx",
  "/boilerplate/app/navigators/DemoNavigator.tsx",
  "/boilerplate/app/navigators/index.ts",
  "/boilerplate/app/navigators/navigation-utilities.ts",
  "/boilerplate/app/screens/DemoCommunityScreen.tsx",
  "/boilerplate/app/screens/DemoDebugScreen.tsx",
  "/boilerplate/app/screens/DemoPodcastListScreen.tsx",
  "/boilerplate/app/screens/LoginScreen.tsx",
  "/boilerplate/app/screens/WelcomeScreen.tsx",
  "/boilerplate/app/screens/index.ts",
  "/boilerplate/app/screens/DemoComponentsScreen/DemoComponentsScreen.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/DemoDivider.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/DemoUseCase.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/DrawerIconButton.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoAutoImage.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoButton.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoHeader.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoIcon.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoListItem.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoText.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/DemoTextField.tsx",
  "/boilerplate/app/screens/DemoComponentsScreen/demos/index.ts",
  "/boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx",
  "/boilerplate/app/screens/ErrorScreen/ErrorDetails.tsx",
  "/boilerplate/app/services/api/api-problem.test.ts",
  "/boilerplate/app/services/api/api-problem.ts",
  "/boilerplate/app/services/api/api.ts",
  "/boilerplate/app/services/api/api.types.ts",
  "/boilerplate/app/services/api/index.ts",
  "/boilerplate/app/services/reactotron/index.ts",
  "/boilerplate/app/services/reactotron/reactotron-client.ts",
  "/boilerplate/app/services/reactotron/reactotron-client.web.ts",
  "/boilerplate/app/services/reactotron/reactotron-config.ts",
  "/boilerplate/app/services/reactotron/reactotron-fake.ts",
  "/boilerplate/app/services/reactotron/reactotron.ts",
  "/boilerplate/app/theme/colors.ts",
  "/boilerplate/app/theme/index.ts",
  "/boilerplate/app/theme/spacing.ts",
  "/boilerplate/app/theme/timing.ts",
  "/boilerplate/app/theme/typography.ts",
  "/boilerplate/app/utils/crash-reporting.ts",
  "/boilerplate/app/utils/delay.ts",
  "/boilerplate/app/utils/ignore-warnings.ts",
  "/boilerplate/app/utils/open-link-in-browser.ts",
  "/boilerplate/app/utils/set-dev.js",
  "/boilerplate/app/utils/storage/index.ts",
  "/boilerplate/app/utils/storage/storage.test.ts",
  "/boilerplate/app/utils/storage/storage.ts",
  "/boilerplate/assets/icons/back.png",
  "/boilerplate/assets/icons/back@2x.png",
  "/boilerplate/assets/icons/back@3x.png",
  "/boilerplate/assets/icons/bell.png",
  "/boilerplate/assets/icons/bell@2x.png",
  "/boilerplate/assets/icons/bell@3x.png",
  "/boilerplate/assets/icons/caretLeft.png",
  "/boilerplate/assets/icons/caretLeft@2x.png",
  "/boilerplate/assets/icons/caretLeft@3x.png",
  "/boilerplate/assets/icons/caretRight.png",
  "/boilerplate/assets/icons/caretRight@2x.png",
  "/boilerplate/assets/icons/caretRight@3x.png",
  "/boilerplate/assets/icons/check.png",
  "/boilerplate/assets/icons/check@2x.png",
  "/boilerplate/assets/icons/check@3x.png",
  "/boilerplate/assets/icons/clap.png",
  "/boilerplate/assets/icons/clap@2x.png",
  "/boilerplate/assets/icons/clap@3x.png",
  "/boilerplate/assets/icons/community.png",
  "/boilerplate/assets/icons/community@2x.png",
  "/boilerplate/assets/icons/community@3x.png",
  "/boilerplate/assets/icons/components.png",
  "/boilerplate/assets/icons/components@2x.png",
  "/boilerplate/assets/icons/components@3x.png",
  "/boilerplate/assets/icons/debug.png",
  "/boilerplate/assets/icons/debug@2x.png",
  "/boilerplate/assets/icons/debug@3x.png",
  "/boilerplate/assets/icons/github.png",
  "/boilerplate/assets/icons/github@2x.png",
  "/boilerplate/assets/icons/github@3x.png",
  "/boilerplate/assets/icons/heart.png",
  "/boilerplate/assets/icons/heart@2x.png",
  "/boilerplate/assets/icons/heart@3x.png",
  "/boilerplate/assets/icons/hidden.png",
  "/boilerplate/assets/icons/hidden@2x.png",
  "/boilerplate/assets/icons/hidden@3x.png",
  "/boilerplate/assets/icons/index.ts",
  "/boilerplate/assets/icons/ladybug.png",
  "/boilerplate/assets/icons/ladybug@2x.png",
  "/boilerplate/assets/icons/ladybug@3x.png",
  "/boilerplate/assets/icons/lock.png",
  "/boilerplate/assets/icons/lock@2x.png",
  "/boilerplate/assets/icons/lock@3x.png",
  "/boilerplate/assets/icons/menu.png",
  "/boilerplate/assets/icons/menu@2x.png",
  "/boilerplate/assets/icons/menu@3x.png",
  "/boilerplate/assets/icons/more.png",
  "/boilerplate/assets/icons/more@2x.png",
  "/boilerplate/assets/icons/more@3x.png",
  "/boilerplate/assets/icons/pin.png",
  "/boilerplate/assets/icons/pin@2x.png",
  "/boilerplate/assets/icons/pin@3x.png",
  "/boilerplate/assets/icons/settings.png",
  "/boilerplate/assets/icons/settings@2x.png",
  "/boilerplate/assets/icons/settings@3x.png",
  "/boilerplate/assets/icons/slack.png",
  "/boilerplate/assets/icons/slack@2x.png",
  "/boilerplate/assets/icons/slack@3x.png",
  "/boilerplate/assets/icons/view.png",
  "/boilerplate/assets/icons/view@2x.png",
  "/boilerplate/assets/icons/view@3x.png",
  "/boilerplate/assets/icons/x.png",
  "/boilerplate/assets/icons/x@2x.png",
  "/boilerplate/assets/icons/x@3x.png",
  "/boilerplate/assets/images/app-icon-all.png",
  "/boilerplate/assets/images/app-icon-android-adaptive-background.png",
  "/boilerplate/assets/images/app-icon-android-adaptive-foreground.png",
  "/boilerplate/assets/images/app-icon-android-legacy.png",
  "/boilerplate/assets/images/app-icon-ios.png",
  "/boilerplate/assets/images/app-icon-web-favicon.png",
  "/boilerplate/assets/images/cr-logo.png",
  "/boilerplate/assets/images/cr-logo@2x.png",
  "/boilerplate/assets/images/cr-logo@3x.png",
  "/boilerplate/assets/images/logo.png",
  "/boilerplate/assets/images/logo@2x.png",
  "/boilerplate/assets/images/logo@3x.png",
  "/boilerplate/assets/images/rnl-logo.png",
  "/boilerplate/assets/images/rnl-logo@2x.png",
  "/boilerplate/assets/images/rnl-logo@3x.png",
  "/boilerplate/assets/images/rnn-logo.png",
  "/boilerplate/assets/images/rnn-logo@2x.png",
  "/boilerplate/assets/images/rnn-logo@3x.png",
  "/boilerplate/assets/images/rnr-logo.png",
  "/boilerplate/assets/images/rnr-logo@2x.png",
  "/boilerplate/assets/images/rnr-logo@3x.png",
  "/boilerplate/assets/images/splash-logo-all.png",
  "/boilerplate/assets/images/splash-logo-android-universal.png",
  "/boilerplate/assets/images/splash-logo-ios-mobile.png",
  "/boilerplate/assets/images/splash-logo-ios-tablet.png",
  "/boilerplate/assets/images/splash-logo-web.png",
  "/boilerplate/assets/images/welcome-face.png",
  "/boilerplate/assets/images/welcome-face@2x.png",
  "/boilerplate/assets/images/welcome-face@3x.png",
  "/boilerplate/bin/downloadExpoApp.sh",
  "/boilerplate/bin/postInstall",
  "/boilerplate/bin/setup",
  "/boilerplate/e2e/README.md",
  "/boilerplate/e2e/config.json",
  "/boilerplate/e2e/firstTest.spec.js",
  "/boilerplate/e2e/init.js",
  "/boilerplate/e2e/reload.js",
  "/boilerplate/ignite/templates/app-icon/android-adaptive-background.png",
  "/boilerplate/ignite/templates/app-icon/android-adaptive-foreground.png",
  "/boilerplate/ignite/templates/app-icon/android-legacy.png",
  "/boilerplate/ignite/templates/app-icon/ios-universal.png",
  "/boilerplate/ignite/templates/component/NAME.tsx.ejs",
  "/boilerplate/ignite/templates/model/NAME.test.ts.ejs",
  "/boilerplate/ignite/templates/model/NAME.ts.ejs",
  "/boilerplate/ignite/templates/navigator/NAME-navigator.tsx.ejs",
  "/boilerplate/ignite/templates/screen/NAME-screen.tsx.ejs",
  "/boilerplate/ignite/templates/splash-screen/logo.png",
  "/boilerplate/ios/.xcode.env",
  "/boilerplate/ios/.xcode.env.local",
  "/boilerplate/ios/HelloWorld-Bridging-Header.h",
  "/boilerplate/ios/Podfile",
  "/boilerplate/ios/Podfile.lock",
  "/boilerplate/ios/Podfile.properties.json",
  "/boilerplate/ios/HelloWorld/AppDelegate.h",
  "/boilerplate/ios/HelloWorld/AppDelegate.mm",
  "/boilerplate/ios/HelloWorld/BootSplash.storyboard",
  "/boilerplate/ios/HelloWorld/Info.plist",
  "/boilerplate/ios/HelloWorld/ignite-noop.swift",
  "/boilerplate/ios/HelloWorld/main.m",
  "/boilerplate/ios/HelloWorld/Images.xcassets/Contents.json",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Contents.json",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-1024-ios-marketing.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-20-ipad.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-20-ipad@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-20-iphone@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-20-iphone@3x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-29-ipad.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-29-ipad@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-29-iphone@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-29-iphone@3x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-40-ipad.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-40-ipad@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-40-iphone@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-40-iphone@3x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-60-iphone@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-60-iphone@3x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-76-ipad.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-76-ipad@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/AppIcon.appiconset/Icon-83.5-ipad@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/BootSplashLogo.imageset/Contents.json",
  "/boilerplate/ios/HelloWorld/Images.xcassets/BootSplashLogo.imageset/bootsplash_logo.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/BootSplashLogo.imageset/bootsplash_logo@2x.png",
  "/boilerplate/ios/HelloWorld/Images.xcassets/BootSplashLogo.imageset/bootsplash_logo@3x.png",
  "/boilerplate/ios/HelloWorld.xcodeproj/project.pbxproj",
  "/boilerplate/ios/HelloWorld.xcodeproj/xcshareddata/xcschemes/HelloWorld.xcscheme",
  "/boilerplate/ios/HelloWorld.xcworkspace/contents.xcworkspacedata",
  "/boilerplate/ios/HelloWorld.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist",
  "/boilerplate/ios/HelloWorldTests/HelloWorldTests.m",
  "/boilerplate/ios/HelloWorldTests/Info.plist",
  "/boilerplate/ios/build/generated/ios/React-Codegen.podspec.json",
  "/boilerplate/ios/build/generated/ios/FBReactNativeSpec/FBReactNativeSpec-generated.mm",
  "/boilerplate/ios/build/generated/ios/FBReactNativeSpec/FBReactNativeSpec.h",
  "/boilerplate/test/i18n.test.ts",
  "/boilerplate/test/mock-async-storage.ts",
  "/boilerplate/test/mock-file.ts",
  "/boilerplate/test/mock-i18n.ts",
  "/boilerplate/test/mock-react-native-image.ts",
  "/boilerplate/test/mock-reactotron.ts",
  "/boilerplate/test/setup.ts",
]
`;
